src/NodeChanger/MethodNameChanger.php

```diff
@@ -10,12 +10,29 @@
 {
     /**
      * @param string[] $renameMethodMap
+     * @param string[]|string[] $acceptedAssertionMethods
      */
-    public function renameNode(MethodCall $node, array $renameMethodMap): ?Node
-    {
+    public function renameNode(
+        MethodCall $node,
+        array $renameMethodMap,
+        array $acceptedAssertionMethods = [],
+        ?string $activeFuncCallName = null
```

Comment:
Null arguments are anti-pattern and very hard to understand from outside.

This should be split into 2 methods, always with all arguments required.